cmake/Modules/FindComputeCpp.cmake

```diff
@@ -308,7 +310,7 @@ function(__build_ir)
 
   if(NOT MSVC)
     # Add a custom target for the generated integration header
-    add_custom_target(${headerTargetName} DEPENDS ${outputSyclFile})
+    add_custom_target(${headerTargetName} DEPENDS ${outputDeviceFile} ${outputSyclFile})
```

Comment:
I think the documentation at the top of the file should be updated referring to the fact that now we output the bc file as well as the sycl one

Reply:
good point, fixed now